src/redturtle/prenotazioni/restapi/serializers/adapters/prenotazione.py

```diff
@@ -100,6 +100,9 @@ def __call__(self, *args, **kwargs):
             "requirements": requirements,
             "modification_date": json_compatible(self.prenotazione.modified()),
             "creation_date": json_compatible(self.prenotazione.created()),
+            "additional_fields": json.loads(
```

Comment:
if you use a JSONfield in prenotazioneType, you don't need this probably

Reply:
done